app/views/automated_tests/_test_script_upload.html.erb

```diff
@@ -25,7 +26,7 @@
     <%= link_to_function (saved ? '[+]' : '[-]'), "toggleSettings(this);", :class => "collapse" %>
   </legend>
 
-  <%= f.hidden_field :seq_num, :value => f.object.seq_num || 1.0, :class => "seqnum" %>
+  <%= f.hidden_field :seq_num, :value => f.object.seq_num || nil, :class => "seqnum" %>
```

Comment:
This isn't good enough, though I'm not sure why.

Use `value: saved ? f.object.seq_num : nil`.